src/cmd-prune

```diff
@@ -31,6 +34,45 @@ strategy.add_argument("--build", metavar="BUILDID", action='append',
                       default=[], help="Explicitly prune BUILDID")
 args = parser.parse_args()
 
+
+def nevra_to_cache_branch(pkg):
+    name = pkg[0]
+
+    epoch = "" if pkg[1] == "0" else pkg[1] + "_3A"
+    version = pkg[2]
+    release = pkg[3]
+    evr = epoch + version + "-" + release
+
+    arch = "x86__64" if pkg[4] == "x86_64" else pkg[4]
```

Comment:
Let's strive to have CoreOS be architecture independent as much as possible - today OpenShift 4/RHCOS ships for [Power](https://en.wikipedia.org/wiki/Power_ISA) and [IBM Z](https://en.wikipedia.org/wiki/Linux_on_IBM_Z) for example.

I think this would be better if we tried to basically re-implement the C logic because that would naturally handle this `_` case in `x86_64` better along with also handling e.g. any packages which happen to have `_` in them.

Something like this?  (not tested)

```
def rpmostree_quote(s):
  r = ""
  for c in s:
    if c in ('.', '_') or c in string.ascii_letters or c in string.digits:
      r += c
    elif c == '_':
      r += "__"
    else:
      r += "{0:02X}".format(ord(c))
  return r
```